dockers/docker-base/Dockerfile.j2

```diff
@@ -34,25 +34,45 @@ COPY ["etc/rsyslog.conf", "/etc/rsyslog.conf"]
 COPY ["etc/rsyslog.d/*", "/etc/rsyslog.d/"]
 COPY ["root/.vimrc", "/root/.vimrc"]
 
+# Install dependencies of supervisor
```

Comment:
"_RDEPENDS" is only used for installing runtime dependencies that _we build_, correct? The packages I am installing here are standard packages in the Debian repo.